Automattic/woocommerce-payments
# Update subscriptions-core to `1.4.0`.

#### Changes proposed in this Pull Request

- Load newer woocommerce-subscriptions-core's version, `1.4.0`.
- Add changes of woocommerce-subscriptions-core to the changelog.
- Updated `composer.lock` as a result of running `composer update woocommerce/subscriptions-core`.

#### Testing instructions

* Checkout this branch.
* [Husky hook should run `composer install`](https://github.com/Automattic/woocommerce-payments/blob/develop/.husky/post-checkout#L8) and your `vendor/woocommerce/subscriptions-core` should be updated to be `1.4.0`. Confirm that [latest changes from `1.4.0`](https://github.com/Automattic/woocommerce-subscriptions-core/commits/1.4.0) is present, for example, [vendor/woocommerce/subscriptions-core/changelog.txt is up-to-date](https://github.com/Automattic/woocommerce-subscriptions-core/blob/1.4.0/changelog.txt).

-------------------


**Post merge**




Closes https://github.com/Automattic/woocommerce-payments/issues/3537.
Closes https://github.com/Automattic/woocommerce-payments/issues/3596.
